src/app/forms/models/async-validators.enum.ts

```diff
@@ -7,5 +7,6 @@ export enum AsyncValidatorNames {
   RequiredIfNotResultAndSiblingEquals = 'requiredIfNotResultAndSiblingEquals',
   RequiredIfNotResult = 'requiredIfNotResult',
   HideIfEqualsWithCondition = 'hideIfEqualsWithCondition',
-  PassResultDependantOnCustomDefects = 'passResultDependantOnCustomDefects'
+  PassResultDependantOnCustomDefects = 'passResultDependantOnCustomDefects',
+  ValidateVin = 'validateVin'
```

Comment:
PART 1:
So this is going to be a multi-part suggestion. I don't think you need this anymore
```suggestion
```